### PR TITLE
Trophies endgame

### PR DESCRIPTION
**Pull request by nilllzz**

### CODE REVIEW

File: src/app/components/trophy/thumbnail/thumbnail.ts [hunk 1]
```diff
@@ -16,10 +16,10 @@ const imgMapping: any = {
 	'platinum-secret': require('./platinum-secret.png'),
 };
 
-export function getTrophyImg(trophy: BaseTrophy, isAchieved = true) {
+export function getTrophyImg(trophy: BaseTrophy) {
 	// Make sure we don't show thumbnails for secret trophies unless they've
 	// been achieved.
-	if (trophy.has_thumbnail && (!trophy.secret || isAchieved || trophy.is_owner)) {
+	if (trophy.has_thumbnail && (!trophy.secret || trophy.is_achieved || trophy.has_perms)) {
```
Comment (YLivay) [inline src/app/components/trophy/thumbnail/thumbnail.ts]: Consider making this a getter on the trophy model for wether the trophy info should show.

File: src/app/components/trophy/thumbnail/thumbnail.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ export function getTrophyImg(trophy: BaseTrophy, isAchieved = true) {
 		img = 'platinum';
 	}
 
-	if (trophy.secret && !isAchieved) {
+	if (trophy.secret && !(trophy.is_achieved || trophy.has_perms)) {
```
Comment (YLivay) [inline src/app/components/trophy/thumbnail/thumbnail.ts]: same comment

File: src/app/components/trophy/thumbnail/thumbnail.ts [hunk 3]
```diff
@@ -56,11 +56,11 @@ export default class AppTrophyThumbnail extends Vue {
 	@Prop(Object)
 	trophy!: BaseTrophy;
 	@Prop(Boolean)
-	isAchieved?: boolean;
-	@Prop(Boolean)
 	noTooltip?: boolean;
 	@Prop(Boolean)
 	noDifficulty?: boolean;
+	@Prop(Boolean)
```
Comment (YLivay) [inline src/app/components/trophy/thumbnail/thumbnail.ts]: put newlines between the props while you're at it, pls thanks

File: src/app/components/trophy/thumbnail/thumbnail.ts [hunk 4]
```diff
@@ -74,12 +74,12 @@ export default class AppTrophyThumbnail extends Vue {
 	get hasThumbnailImg() {
 		return (
 			this.trophy.has_thumbnail &&
-			(!this.trophy.secret || this.isAchieved || this.trophy.is_owner)
+			(!this.trophy.secret || this.trophy.is_achieved || this.trophy.has_perms)
```
Comment (YLivay) [inline src/app/components/trophy/thumbnail/thumbnail.ts]: here too

File: src/app/components/trophy/thumbnail/thumbnail.ts (hunk 2 above)
Comment (nilllzz) [reply src/app/components/trophy/thumbnail/thumbnail.ts]: can't do this here, condition is slightly different.

**Pull request merged**